scripts/request_withdraw.js

```diff
@@ -0,0 +1,33 @@
+const SnappBase = artifacts.require("SnappBase")
+const zero_address = 0x0
+
+module.exports = async (callback) => {
+  try {
+    const arguments = await process.argv.slice(4)
+    if (arguments.length != 3) {
```

Comment:
Can you please make that work so that it works with all networks? You can/should simply use the getParameterHelper from my last PR.

Reply:
👍

Reply:
Right. Has this already been Merged? I was kinda expecting to see this when I copied and pasted from the deposit script.

Reply:
I have merged it some hours ago. It's in there...